test/spec/update-listeners.js

```diff
@@ -303,6 +303,23 @@ describe('update-listeners', () => {
     })
   })
 
+  it('should handle dbnames with slashes', async() => {
+    await createListeners()
+
+    let DBNameWithSlashes = {}
```

Comment:
please rename the variables to `dbNameWithSlashes` and `dbNameWithoutSlashes`

Reply:
Done.